src/chainstate/burn/db/sortdb.rs

```diff
@@ -91,19 +93,19 @@ pub const REWARD_WINDOW_END: u64 = 144 * 90 + REWARD_WINDOW_START;
 
 pub type BlockHeaderCache = HashMap<ConsensusHash, (Option<BlockHeaderHash>, ConsensusHash)>;
 
-// for using BurnchainHeaderHash values as block hashes in a MARF
-impl From<BurnchainHeaderHash> for BlockHeaderHash {
-    fn from(bhh: BurnchainHeaderHash) -> BlockHeaderHash {
-        BlockHeaderHash(bhh.0)
-    }
-}
+// // for using BurnchainHeaderHash values as block hashes in a MARF
```

Comment:
Is there a reason these aren't being used anymore?

Reply:
Yes -- these implementations, if required, need to moved to `stacks-common`, because this is a foreign trait that is being defined for (now) foreign types. The "foreign trait implementation" constraint in rust is one of the biggest things that necessitates moving implementations around in this PR (e.g., it's why a lot of the codec implementations moved to stacks-common).

Reply:
Removed the commented out region